src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+import javax.swing.*;
+
+public class Square extends Figure {
+    private double side;
+
+    public Square(double side, String color) {
+        super(color);
+        this.side = side;
+    }
+
+    public double getSide() {
+        return side;
+    }
+
+    public void setSide(double side) {
+        this.side = side;
+    }
+
+    @Override
+    public double area() {
+        return side * side;
```

Comment:
The method name 'area' should be a verb or a verb phrase, something like 'calculateArea' would be more appropriate.